src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,67 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-8#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+      = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+    .comment_new.mt-3
+      = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+  .col-md-4#side-links
```

Comment:
To me all the side links don't belong together with the reviews. Please separate those and find a more useful place.

Reply:
I've removed the superseders info from sidelinks, which is moved with the created_at message right below the title. A new card is added to the backlog to deal with maintainers' info. https://trello.com/c/Gg6UnB4W